packages/react-slider/src/components/Slider/useSlider.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react';
+import { makeMergeProps, resolveShorthandProps, useId, useControllableValue } from '@fluentui/react-utilities';
+import {
+  getCode,
+  ArrowDownKey,
+  ArrowUpKey,
+  ArrowLeftKey,
+  ArrowRightKey,
+  PageDownKey,
+  PageUpKey,
+  HomeKey,
+  EndKey,
+} from '@fluentui/keyboard-key';
+import { on } from '@fluentui/utilities';
```

Comment:
We should not be using `@fluentui/utilities` in converged components. You might want to copy the implementation of `on` into `@fluentui/react-utilities` and importing this from there instead.